modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -120,11 +114,9 @@ spec:
    matchLabels:
      pools.operator.machineconfiguration.openshift.io/worker: '' <1>
  containerRuntimeConfig: <2>
-   pidsLimit: 2048
    logLevel: debug
    overlaySize: 8G
-   logSizeMax: "-1"
-----
+   ----
```

Comment:
If you remove the leading spaces before `----`, the build should pass.

Reply:
Thank you for reviewing, now the build has passed